lib/js/components/Icons/FeatureIcon/FeatureIcon.vue

```diff
@@ -0,0 +1,182 @@
+
+<style lang="scss">
+@import '../../../../styles/components/icons';
+@import '../../../../styles/settings/spacings';
+@import '../../../../styles/settings/colors/tokens';
+
+$feature-icon-colors: (
+	'neutral': (
+		'background': $color-neutral-background-medium,
+		'border': $color-neutral-background,
+		'icon': $color-neutral-icon,
+	),
+	'neutral-weak': (
+		'background': $color-neutral-background-medium,
+		'border': $color-neutral-background,
+		'icon': $color-neutral-icon-weak,
+	),
+	'primary': (
+		'background': $color-primary-background-medium,
+		'border': $color-primary-background,
+		'icon': $color-primary-icon,
+	),
+	'success': (
+		'background': $color-success-background-medium,
+		'border': $color-success-background,
+		'icon': $color-success-icon,
+	),
+	'danger': (
+		'background': $color-danger-background-medium,
+		'border': $color-danger-background,
+		'icon': $color-danger-icon,
+	),
+	'warning': (
+		'background': $color-warning-background-medium,
+		'border': $color-warning-background,
+		'icon': $color-warning-icon,
+	),
+	'info': (
+		'background': $color-info-background-medium,
+		'border': $color-info-background,
+		'icon': $color-info-icon,
+	),
+);
+
+$feature-icon-padding-large: 10px;
+
+@mixin setBorderColor($border) {
+	border-color: $border;
+}
+
+@mixin setFeatureIconColor($self, $background, $icon) {
+	background-color: $background;
+	border-color: $background;
+
+	#{$self}__icon {
+		color: $icon;
+	}
+}
+
+.featureIcon {
+	$self: &;
+
+	@each $color-name, $color-map in $feature-icon-colors {
+		&.-#{$color-name} {
+			@include setFeatureIconColor(
+				$self,
+				map-get($color-map, 'background'),
+				map-get($color-map, 'icon')
+			);
+		}
+		&.-#{$color-name}.-doubleBackground {
+			@include setBorderColor(map-get($color-map, 'border'));
+		}
+	}
+
+	border-radius: 100%;
+	border-style: solid;
+	border-width: $space-xxxs;
+	display: inline-block;
+	padding: $space-xxxs;
+
+	&.-small {
+		border-width: $space-xxxxs;
+		padding: $space-xxxxs;
+	}
+
+	&.-large {
+		border-width: $space-xxxs;
+		padding: $feature-icon-padding-large;
+	}
+
+	&.-x-large {
+		border-width: $space-xxs;
+		padding: $space-xs;
+	}
+}
+</style>
+
+<script lang="ts">
+import { VueConstructor } from 'vue';
+import WnlIcon, { ICONS, ICON_SIZES } from '../Icon';
+import { FEATURE_ICON_SIZES, FEATURE_ICON_COLOR } from './FeatureIcon.consts';
+
+export default {
+	name: 'FeatureIcon',
+	components: {
+		WnlIcon,
+	},
+	props: {
+		icon: {
+			type: Object,
+			default: null,
+			validator(icon: VueConstructor) {
+				return Object.values(ICONS).includes(icon);
+			},
+		},
+		size: {
+			type: String,
+			default: FEATURE_ICON_SIZES.MEDIUM,
+			validator: (value: string) => Object.values(FEATURE_ICON_SIZES).includes(value),
+		},
+		color: {
+			type: String,
+			required: true,
+			validator(color) {
+				return Object.values(FEATURE_ICON_COLOR).includes(color);
+			},
+		},
+		doubleBackground: {
+			type: Boolean,
+			default: true,
```

Comment:
according to agreement with UX, it should be `false`

Reply:
Fixed.